src/pages/dashboard/index.vue

```diff
@@ -272,7 +272,12 @@ export default {
         1
       );
       this.isOpenFollowersDialog = true;
-      await this.fetchFollowers();
+      try {
+        await this.walletFetchFollowers();
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error(error);
+      }
```

Comment:
In this same commit, `walletFetchFollowers` has a huge try catch block that catches any axios error
Do we really need this try catch?